src/VisualStudio/Core/Def/Implementation/AbstractEditorFactory.cs

```diff
@@ -129,14 +129,13 @@ int IVsEditorFactory.Close()
 
                     // We must create the WinForms designer here
                     var loaderName = GetWinFormsLoaderName(vsHierarchy);
```

Comment:
❓ Do we still need this call, or will the logic to use .NET Framework vs. .NET Core be moving at the same time?

https://github.com/dotnet/roslyn/blob/8fca76de67f88ed0c3dc69c09ae3bba6fa691d70/src/VisualStudio/Core/Def/Implementation/AbstractEditorFactory.cs#L178-L213

Reply:
I'm pretty sure we still need to support both. This is just so we can do something in the .net framework designer (detect if the .net core vsix is installed and pop a gold bar if it's not) before the code view is shown.

Reply:
Yes, we still need this for now. Once the new WinForms designer is in-box, we'll remove this logic. The key issue is a bit of a chicken-and-egg problem. Because the `IVsHierarchy` isn't passed to designer code until after the `IVSMDDesignerLoader` is created, we don't yet have a way to detect the TFM on the designer side at the right time.